docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -7,37 +7,37 @@ All gateway events in Discord are tagged with an opcode that denotes the payload
-| 4014 | disallowed intent(s)  | You sent a disallowed intent for a [Gateway Intent](#DOCS_TOPICS_GATEWAY/gateway-intents). You may have tried to specify an intent that you [have not enabled or are not whitelisted for](#DOCS_TOPICS_GATEWAY/privileged-intents)                                           |
+| ---- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| 4000 | Unknown error         | We're not sure what went wrong. Try reconnecting?                                                                                                                                                                                                                               |
+| 4001 | Unknown opcode        | You sent an invalid [Gateway opcode](#DOCS_TOPICS_GATEWAY/payloads-and-opcodes) or an invalid payload for an opcode. Don't do that!                                                                                                                                       |
+| 4002 | Decode error          | You sent an invalid [payload](#DOCS_TOPICS_GATEWAY/sending-payloads) to us. Don't do that!                                                                                                                                                                                     |
+| 4003 | Not authenticated     | You sent us a payload prior to [identifying](#DOCS_TOPICS_GATEWAY/identify).                                                                                                                                                                                                   |
+| 4004 | Authentication failed | The account token sent with your [identify payload](#DOCS_TOPICS_GATEWAY/identify) is incorrect.                                                                                                                                                                           |
+| 4005 | Already authenticated | You sent more than one identify payload. Don't do that!                                                                                                                                                                                                                         |
+| 4007 | Invalid `seq`         | The sequence sent when [resuming](#DOCS_TOPICS_GATEWAY/resume) the session was invalid. Reconnect and start a new session.                                                                                                                                                   |
+| 4008 | Rate imited           | Woah nelly! You're sending payloads to us too quickly. Slow it down! You will be disconnected on receiving this.                         |
```

Comment:
```suggestion
| 4008 | Rate limited           | Woah nelly! You're sending payloads to us too quickly. Slow it down! You will be disconnected on receiving this.                         |
```

Reply:
Good spot, I'll fix that immediately. Thanks :)